Dockerfile

```diff
@@ -68,6 +68,7 @@ WORKDIR /home/jekyll
 # kabanero.io gem dependencies
 COPY ./scripts /home/jekyll/scripts
 COPY Gemfile /home/jekyll
+COPY Gemfile.lock /home/jekyll
```

Comment:
to reduce the size of the image you can do this COPY in 1 layer like:

`Gemfile Gemfile.lock /home/jekyll`